### PR TITLE
Update pr-bot to send PRs to new repo

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Update the PR-bot to send pull-requests to the new manifests repository.

Demo PR: https://github.com/cds-snc/notification-manifests/pull/16

:warning: **This shoud not be released before next Monday!**

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -136,35 +131,35 @@ async function run() {
       const apiMsgs = await getCommitMessages("notification-api", oldApiSha)
 
       let logs = `ADMIN: \n\n ${adminMsgs.join("\n")} \n\n API: \n\n ${apiMsgs.join("\n")}`
-      if (await hasChangesOnStagingTfRepo()) {
-        logs = `⚠️ There was a merged pull request in [notification-staging-tf](https://github.com/cds-snc/notification-staging-tf) since last deploy in production! Check first that you don't need to change the infrastructure here. \n\n ${logs}`
+      if (await isNotLatestVersion()) {
+        logs = `⚠️ The production version of manifests is behind the latest staging version. Consider upgrading to the latest version before! \n\n ${logs}`
```
Comment (brdunfield) [inline dist/index.js:135]: Suggestion:
Consider upgrading to the latest version before {merging this pull request}.

Comment (AntoineAugusti) [reply dist/index.js]: Cheers, just changed this

**Review by brdunfield — APPROVED** [review]
🦑

**Comment by AntoineAugusti** [comment]
@brdunfield If you wanna double-check the new changes.

I just remembered that we know need to check for manifests + Terraform versions. Both checks will give us increased security.

**Comment by brdunfield** [comment]
Good catch. Looking good!

**Pull request merged**